Sources/ArcGISToolkit/Components/FeatureFormView/AttachmentHelpers/AttachmentCameraController.swift

```diff
@@ -98,7 +98,7 @@ class AttachmentUIImagePickerController: UIImagePickerController {
     override func viewDidAppear(_ animated: Bool) {
         cameraCaptureModeObserver = NotificationCenter.default.addObserver(forName: NSNotification.Name("SourceFormatDidChange"), object: nil, queue: nil) { _ in
             Task {
-                await self.action?(self.cameraCaptureMode)
+                self.action?(self.cameraCaptureMode)
```

Comment:
Xcode 15.4
![image](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/eb3a68ae-2c84-452a-a1f2-d107febfd481)

Xcode 16
![image](https://github.com/Esri/arcgis-maps-sdk-swift-toolkit/assets/16397058/bfa9a092-cc4b-4d87-a24f-3a44920f460c)

I'm not sure I understand why 16.0 considers the expression async while 15.4 doesn't. 

Since we're not using Swift 6 yet, I assume the proper thing to do is remove `await` for now and revisit this when we move to Swift 6?

Reply:
You could do that. But if you want to, you could account for both:

```swift
#if compiler(>=6.0)
// Use await.
#else
// Don't use await.
#endif
```